nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/util/PackageValidator.java

```diff
@@ -25,9 +25,9 @@
 public final class PackageValidator
 {
   public static final String NOT_VALID_PATH_ERROR_MESSAGE =
-      "Not a valid upload path. Should be e.g. src/contrib or bin/<os>/contrib/<R_version>.";
+      "Provided path is not valid and is expecting src/contrib or bin/<os>/contrib/<R_version>";
 
-  public static final String NOT_VALID_EXTENSION_ERROR_MESSAGE = "Extension not .zip, .tar.gz or .tgz.";
+  public static final String NOT_VALID_EXTENSION_ERROR_MESSAGE = "Provided extension is not .zip, tar.gz or .tgz";
```

Comment:
Trivial: It is trivally weird to me that some of the extensions start with periods and others don't.  I trust y'all are more familiar with conventions than I am (non-blocker in any case).
(Sorry @aornatovskyy I didn't notice this when we spoke)

Reply:
Oh, didn't notice. Just copied from ticket. Updated in 13afaf2e8d689d64dac4803a48a53927498645c9